python/ray/autoscaler/autoscaler.py

```diff
@@ -140,13 +141,16 @@ def _update(self):
             return
 
         self.last_update_time = now
-        nodes = self.workers()
+        all_nodes = self.workers()
+        nodes = self.managed_workers()
```

Comment:
can we rename this variable `managed_worker_nodes`?

Reply:
@ericl  just recommended I not change the variable names to keep the diff small. 

Personally I prefer more accurate variable names, but I'd rather get this merged than argue about that.